lib/mongo/error/operation_failure.rb

```diff
@@ -162,6 +168,16 @@ def change_stream_resumable_code?
       end
       private :change_stream_resumable_code?
 
+      def change_stream_resumable_label?
```

Comment:
This method would return true when there is no label, which I think is confusing. Can it be changed to `change_stream_not_resumable_label?` which would return true if 1) there is a label and 2) it is a change stream not resumable one?